### ISSUE TITLE
E121, E126 false positives

### ISSUE THREAD

**Issue by jianli** [PyCQA/pycodestyle#138]
In pep8 v.1.3.3, these two cases are flagged with

> E126 continuation line over-indented for hanging indent

``` python
{
    1:
        999999 if True
        else 0,
}
```

``` python
[
    12,  # this is a multi-line inline                                                                                                                                                                           
         # comment                                                                                                                                                                                               
]
```

and the latter also triggers:

> E121 continuation line indentation is not a multiple of four
